automatic/git.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,96 @@
+$ErrorActionPreference = 'Stop'
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$filePath32 = "$toolsDir\Git-2.11.0-32-bit.exe"
+$filePath64 = "$toolsDir\Git-2.11.0-64-bit.exe"
+$fileArgs = $(
+  '/VERYSILENT /NORESTART /NOCANCEL /SP- ' +
+  '/COMPONENTS="icons,icons\quicklaunch,ext,ext\shellhere,ext\guihere,assoc,assoc_sh" /LOG'
+)
+
+# Parse package parameters
+$pp = Get-PackageParameters
+
+$registryKeyName = 'Git_is1'
```

Comment:
https://github.com/pascalberger/chocolatey-coreteampackages/blob/ff38b7dd3ed75ba574218cc41ab5a823901f690f/automatic/git.install/tools/chocolateyInstall.ps1#L14-L48

I would move this to `helper.ps1`. It defocuses from the task in question with verbosity and details. For example, you might call it like this `Set-GitExtraPararameters $pp` because as far as I can see those are not supported via silent options. Or maybe `Set-GitRegistryOptions $pp`.

Also

https://github.com/pascalberger/chocolatey-coreteampackages/blob/ff38b7dd3ed75ba574218cc41ab5a823901f690f/automatic/git.install/tools/chocolateyInstall.ps1#L52-L61

could perhaps be named `Remove-QuickLaunch4SystemUser`.

https://github.com/pascalberger/chocolatey-coreteampackages/blob/ff38b7dd3ed75ba574218cc41ab5a823901f690f/automatic/git.install/tools/chocolateyInstall.ps1#L63-L67

could be `Stop-GitSSHAgent`

You can also incorporate `Get-AppInstallLocation 'Git version*'` instead of handling registry directly via Git_is1'